knip.json

```diff
@@ -10,7 +10,7 @@
     }
   },
   "ignore": ["**/*.d.ts", "**/dist"],
-  "ignoreWorkspaces": ["packages/artifacts"],
+  "ignoreWorkspaces": ["packages/artifacts", "apps/kudosbot-server"],
```

Comment:
question: why? I wonder if the reasoning would make sense to include as a comment (somehow) inline here?

Reply:
Because kudosbot-server didn't have an entry point. I see that at some point it has actually be removed entirely from the repo, which makes this line redundant.